README.md

```diff
@@ -99,7 +99,6 @@ $ fex2 template splash
 $ fex2 install splash
 $ cat <<EOF >> build_types/gcc_optimized.mk
 include gcc_native.mk
-include common.mk
```

Comment:
Yes, this line is indeed redundant, but I'd leave this it be, just for clarity.

Reply:
Ok :+1: